src/cli/cli_mac_filter.cpp

```diff
@@ -156,21 +195,80 @@ template <> otError MacFilter::Process<Cmd("addr")>(Arg aArgs[])
             SuccessOrExit(error = otLinkFilterAddRssIn(GetInstancePtr(), &extAddr, rss));
         }
     }
+    /**
+     * @cli macfilter addr remove
+     * @code
+     * macfilter addr remove 0f6127e33af6b402
+     * Done
+     * @endcode
+     * @cparam macfilter addr remove @ca{extaddr}
+     * @par
+     * Is available when OPENTHREAD_CONFIG_MAC_FILTER_ENABLE configuration is enabled.
+     * @par
+     * This command removes the specified extended address from the mac filter list.
```

Comment:
When using "mac" in the text we should use MAC (capitalized).

So "MAC filter list", "MAC address", etc

Can you update across all commands here as needed